js/bin/integration.js

```diff
@@ -30,7 +30,7 @@ const {
     Table,
     RecordBatchReader,
     util: { createElementComparator }
-} = require('../targets/apache-arrow/Arrow.es5.min');
+} = require('../targets/apache-arrow/Arrow.es2015.min');
```

Comment:
I think we're using the ES5 bundle here because CI can potentially have (or used to have?) an old version of node.

Reply:
Would that mean we need to build es5 and es2015 bundles in the arrow package? 

I would hope that our CI doesn't use a version that only supports es5 anymore. I'd say we should update the CI if it does.

Reply:
Not sure if it still does. I think the integration tests install node via conda? If this passes CI, then great. If not, we can change this to `../targets/es5/umd/Arrow` instead.

Reply:
It's supposed to use node 14: https://github.com/apache/arrow/blob/113a515e2b84547e5b198fa5d0c489bf363ea46b/ci/docker/conda-integration.dockerfile#L24

The test failure looks unrelated again.

<img width="1299" alt="Screen Shot 2021-04-09 at 14 13 52" src="https://user-images.githubusercontent.com/589034/114241444-fbb61100-993d-11eb-9cf5-2b2d9298793a.png">

Reply:
It looks like the problem is actually that the es2015 bundles expect `window` to be defined. I switched to using cjs.

Reply:
Works now.